src/operators/ExtrudeStacked/ExtrudeStacked.code

```diff
@@ -0,0 +1,331 @@
+Function: InsertVariable
+Declaration: void InsertVariable(const std::string &variableName_);
+Definition:
+// ****************************************************************************
+// Method: ParallelCoordinatesAttributes::InsertVariable
+//
```

Comment:
Please change all Method comments in this file.
1)  Change 'ParallelCoordinatesAttributes' to ExtrudeStackedAttributes
2)  Change Programmer to you
3)  Use current Creation dates
4)  Remove all Modification comments,
5)  Change Note to reflect where you pulled the code from (eg ParallelCoordinates, or Extrude or wherever it was copied from)

Reply:
Done in Qvis and avt